src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -482,6 +482,7 @@ class GroupReadsByUmi
   @arg(flag='T', doc="The output tag for UMI grouping.") val assignTag: String = "MI",
   @arg(flag='m', doc="Minimum mapping quality for mapped reads.")         val minMapQ: Int      = 1,
   @arg(flag='n', doc="Include non-PF reads.")            val includeNonPfReads: Boolean = false,
+  @arg(flag='N', doc="Include UMIs which contain non-ATCG bases (N) in output.")             val includeNonATCG: Boolean = false,
```

Comment:
I don't think we are supporting non-ACGT, for example IUPAC codes.  We're treating Ns as "fifth" base (i.e. an N mismatches an A), so we need to add that to the docstring.  Also see how we calculate the length of the UMI based on how many `[ACGTN]`s there are (not non-ATCGs).   Finally the usage needs updating (see `number of [ACGT]...` and `non-ACGT characters)`).
```suggestion
  @arg(flag='N', doc="Allow UMIs which contain Ns.  Ns will be treated as mismatches to all other non-Ns") val includeNs: Boolean = false,
```